.swiftlint.yml

```diff
@@ -65,7 +65,9 @@ opt_in_rules: # some rules are only opt-in
   # swiftlint rules
   - anyobject_protocol
   - array_init
-  - attributes
+  - attributes:
+      always_on_line_above:
+        - '@Published'
```

Comment:
@cyndichin Lastet version of [swiftlint](https://github.com/realm/SwiftLint/releases/tag/0.51.0) causes 100+ violations without this change, do we need to update to match the rest of Mozilla?

Reply:
let me follow up with Roux on this because I don't see `attributes` in their `.swiftlint.yml` file, but it's marked as Done in the Mozilla SwiftLint spreadsheet. I was going to create a PR with the updated rules next Monday.

I'm also not seeing the violations. Do you have a screenshot of what it looks like?

Reply:
@cyndichin update to the latest swiftlint `brew update && brew upgrade swiftlint`

Reply:
ah okay, I see it. Roux says it has not been incorporated in their main branch yet but they did not have to add the additional field. Most likely due to us using a lot of @Published property wrappers. I'll make sure to check this when updating the rules - thanks!